libs/dh/admin/feature-user-management/src/lib/drawer/tabs/content/dh-user-roles.component.html

```diff
@@ -0,0 +1,62 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'admin.userManagement.tabs.roles'">
+  <ng-container
+    *rxLet="
+      userRoleView$;
+      let userRoleView;
+      error: error;
+      suspense: suspense;
+      errorTrigger: hasGeneralError$
+    "
+  >
+    <watt-card>
+      <watt-card-title>
+        <h3>
+          {{ userRoleView | userRoleCount }}
+        </h3>
+      </watt-card-title>
+      <div class="userRoles" *ngFor="let org of userRoleView.organizations">
+        <ng-container *ngFor="let actor of org.actors">
+          <h6>
+            {{ org?.name }} | GLN {{ actor.actorNumber }} |
+            {{ actor.userRoles | joinUserRoles }}
+          </h6>
+          <mat-divider></mat-divider>
+          <ul class="watt-text-s">
+            <li *ngFor="let userRole of actor.userRoles">
+              <div>{{ userRole.name }}</div>
+              <div>Manglede beskrivelse</div>
```

Comment:
suggestion: move to translation

Reply:
It is a missing field that needs to be provided by the backend, So dont see the need for a translation. I can add a TODO to it?

Reply:
Then it's fine to stay as is until the missing field is added.